code/modules/recycling/conveyor2.dm

```diff
@@ -113,12 +122,32 @@
 				step(A,movedir)
 		CHECK_TICK
 
+
+
 // attack with item, place item on conveyor
-/obj/machinery/conveyor/attackby(var/obj/item/I, mob/user, params)
-	if(isrobot(user))	return //Carn: fix for borgs dropping their modules on conveyor belts
-	user.drop_item()
-	if(I && I.loc)	I.loc = src.loc
-	return
+/obj/machinery/conveyor/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/crowbar))
+		if(!(stat & BROKEN))
+			var/obj/item/conveyor_construct/C = new/obj/item/conveyor_construct(src.loc)
+			C.id = id
+			transfer_fingerprints_to(C)
+		user << "<span class='notice'>You remove the conveyor belt.</span>"

```

Comment:
So in this case: `to_chat(user, "<span class='notice'>You remove the conveyor belt.</span>")` ?